### PR TITLE
List.ItemMetaText for non-icon ItemMeta

### PR DESCRIPTION
**Pull request by fopina**
As discussed in #1337

### CODE REVIEW

**Comment by fopina** [comment]
will do @prateekbh but do you agree with the change as is? or do you prefer some other approach?

File: ts/List/index.tsx [hunk 1]
```diff
@@ -65,6 +65,25 @@ export class ListItemMeta extends ListItemGraphic {
...
 
+export interface IListItemMetaTextProps {}
+export interface IListItemMetaTextState {}
+
+export class ListItemMetaText extends MaterialComponent<
+  IListItemMetaTextProps,
+  IListItemMetaTextState
+> {
+  protected componentName = 'list-item__meta';
+  protected mdcProps = [];
```
Comment (prateekbh) [inline ts/List/index.tsx]: this needs a `padded` option

**Comment by prateekbh** [comment]
The PR looks good, just a small change

File: ts/List/index.tsx (hunk 1 above)
Comment (fopina) [reply ts/List/index.tsx]: could you elaborate? as I'm kind of new to both material, preact and preact-material I was looking for `padded` in whole project to implement the same option but couldn't find anything...

**Comment by artipost** [comment]
Failed pictures (2): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/63bdb86339f91d42d8a11af64db668f4/raw/e2a8c41f4c06d73fd1942730d29b1552f7a04cf8/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [598173755](https://travis-ci.org/prateekbh/preact-material-components/builds/598173755), for commit [d3f8ad4](https://github.com/prateekbh/preact-material-components/pull/1338/commits/d3f8ad4c0bf81b9338dbbf44570d397f13498ce9).

**Comment by fopina** [comment]
docs updated, but no clue on `padded` option (added reply to your comment)

File: ts/List/index.tsx (hunk 1 above)
Comment (prateekbh) [reply ts/List/index.tsx]: woops sorry my bad, was looking at divider--paded

**Comment by prateekbh** [comment]
@fopina this looks good please add the failed pictures here in your repo: https://github.com/prateekbh/preact-material-components/tree/master/docs/tests/golden/wide

**Comment by artipost** [comment]
Failed pictures (1): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/b6fbdbc560c7150c4a3b54969c08b6c9/raw/d7ad3ff563549ab979184d11c165308da072bef6/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [598558074](https://travis-ci.org/prateekbh/preact-material-components/builds/598558074), for commit [e272410](https://github.com/prateekbh/preact-material-components/pull/1338/commits/e27241077b0648a949dae6b69c3cd10e99c069a7).

**Comment by fopina** [comment]
hope it looks good now :+1:

**Pull request merged**